tests/src/Simple/PInvoke/no_unix

```diff
@@ -0,0 +1 @@
+Doesn't work on OSX.
```

Comment:
I have not tested it in Non-windows platform yet. It was failing on OSX. I'll fix it on the next PR.